be/log_discard.h

```diff
@@ -115,8 +115,14 @@ struct m0_be_log_discard {
 	 * XXX Temporary solution.
 	 */
 	struct m0_semaphore                  lds_discard_wait_sem;
-	int                                  lds_discard_left;
 	bool                                 lds_discard_waiting;
+
+
+	/**
+	 * Add single ast to discard multiple ldi
```

Comment:
Please write a better comment, try to reuse previous:

`To run ldsc_discard callback outside the m0_be_log_discard lock.`